batch/test/test_batch.py

```diff
@@ -727,14 +727,19 @@ def test_cant_submit_to_default_with_other_ns_creds(client):
     else:
         assert status['state'] == 'Failed', str(status)
 
+    if DOMAIN.startswith('internal'):
+        without_internal = DOMAIN.split('.', maxsplit=1)[1]
+    else:
+        without_internal = DOMAIN
     builder = client.create_batch()
     j = builder.create_job(
         os.environ['HAIL_HAIL_BASE_IMAGE'],
         [
             '/bin/bash',
             '-c',
             f'''
-jq '.domain = "batch.{DOMAIN}"' /deploy-config/deploy-config.json > /deploy-config/deploy-config.json
+jq '.domain = "{without_internal}"' /deploy-config/deploy-config.json > tmp.json
```

Comment:
Hmm. Yeah, I'm not sure I understand why we had `batch....`, this should be the domain suffix for the current Hail instance. That should always be `hail.is` except in the Aussie's setup.